src/context.rs

```diff
@@ -2430,6 +2430,20 @@ impl ContextWriter {
 
     /* TODO: Handle single reference frame extension */
 
+    // clamp mvs
```

Comment:
I think motion_estimation() can limit the mv range?

Reply:
Not sufficiently, because the range is not the same for all blocks. NEAREST may produce an MV that was ok for a previous block but not the current one.